src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -475,6 +540,136 @@ const ManifestComponent = ({
+        }
+
+        return (
+            <InfoColourBar
+                message={errorText}
+                classname="w-100 m-0 code-editor__error dc__no-border-radius dc__no-top-border dc__no-left-border dc__no-right-border dc__word-break"
+                Icon={ICErrorExclamation}
+                iconClass="icon-dim-16"
+                linkClass="dc__truncate--clamp-6"
+            />
+        )
+    }
+
+    const renderContent = () => {
+        if (!isReadOnlyView && manifestFormConfigurationType === ConfigurationType.GUI) {
+            return (
+                <>
+                    {renderEditorInfo()}
+                    {renderErrorBar()}
+                    <ManifestGUIView
+                        guiSchema={guiSchema}
+                        handleChange={handleGUIViewValueChange}
+                        // For uniformity have called method but as of now in this case it will always be trimedManifestEditorData
+                        manifestYAMLString={getCodeEditorValue()}
+                        handleSwitchToYAMLMode={handleSwitchToYAMLMode}
+                    />
+                </>
+            )
+        }
+
+        return (
+            <CodeEditor
+                defaultValue={showManifestCompareView && desiredManifest}
+                cleanData={showManifestCompareView}
+                diffView={showManifestCompareView}
+                theme="vs-dark--dt"
+                height={isResourceBrowserView ? 'calc(100vh - 119px)' : 'calc(100vh - 77px)'}
+                value={getCodeEditorValue()}
+                mode={MODES.YAML}
+                readOnly={isReadOnlyView}
+                onChange={handleEditorValueChange}
+                loading={loading}
+                customLoader={
+                    <MessageUI
+                        msg={loadingMsg}
+                        icon={MsgUIType.LOADING}
+                        size={24}
+                        minHeight={isResourceBrowserView ? 'calc(100vh - 151px)' : ''}
+                    />
+                }
+                focus={isEditMode}
+            >
+                {renderEditorInfo(true)}
+
+                {!loading &&
+                    !error &&
+                    isConfigDriftEnabled &&
+                    'hasDrift' in _selectedResource &&
+                    _selectedResource.hasDrift &&
+                    !showManifestCompareView &&
+                    renderOutOfSyncWarning &&
+                    renderOutOfSyncWarning(handleDesiredManifestOpen)}
+                {showManifestCompareView && (
+                    <CodeEditor.Header hideDefaultSplitHeader className="p-0">
+                        <div className="dc__split-header">
+                            <div className="dc__split-header__pane flexbox dc__align-items-center dc__content-space dc__gap-8">
+                                <span>Desired manifest</span>
+                                <button
```

Comment:
suggestion: can use Button component

Reply:
Not sure how this would like since there are other buttons in surrounding like renderOutOfSyncWarning, etc